assets/js/modules/analytics/common/create-account-field.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * CreateAccountField component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	Input,
+	TextField,
+} from '../../../material-components';
+import classnames from 'classnames';
+
+const CreateAccountField = ( {
+	validationIssues,
+	setValidationIssues,
+	objectValue,
+	objectSetter,
```

Comment:
These two names sound a bit odd. Why not just `value` and `setValue`?

Reply:
Naming is hard, I have no idea why I came up with those names. Your suggestion is much better, changed in https://github.com/google/site-kit-wp/pull/1389/commits/b12378a778fb80f349d5e670d860a9dc71dbc808.